contrib/gcppubsub/config/300-gcppubsubsource.yaml

```diff
@@ -51,6 +51,8 @@ spec:
               type: string
             sink:
               type: object
+            transformer:
```

Comment:
Could you please add description fields for all these while you're here? Something like this:
https://github.com/vaikas-google/gcs/blob/master/config/gcs.yaml#L36

Reply:
Done, thanks!

Reply:
This is awesome, thanks for doing it!